Dockerfile

```diff
@@ -0,0 +1,213 @@
+# Generated by Neurodocker and Reproenv.
+
+FROM docker.io/gentoo/portage:20240324 as portage
+FROM docker.io/gentoo/stage3:20240318
+
+COPY --from=portage /var/db/repos/gentoo /var/db/repos/gentoo
+
+ARG gentoo_hash=2d25617a1d085316761b06c17a93ec972f172fc6
+ARG science_hash=73916dd3680ffd92e5bd3d32b262e5d78c86a448
```

Comment:
@TheChymera could you please provide new hashes needed here for the new/fixed up build and also add a patch for AFNI to reflect change like https://github.com/afni/afni/pull/650/files ? or could you just incorporate it to AFNI package so we could build/test?

Reply:
I was planning to, but after testing the fix doesn't seem to work. collecting more logs so I can report in the upstream issue exactly what's going on how it was built etc.

Reply:
PR here (tested with this repo, for some reason the image takes a long time to build compared to the neurodocker test) → https://github.com/yarikoptic/dsst-defacing-pipeline/pull/1
Commot on the gentoo+neurodocker PR: https://github.com/ReproNim/neurodocker/pull/611/commits/758951bde08016f94ec3027cab32f9b25ce2bdaf